src/entryPoints/HyperSwitchApp.res

```diff
@@ -460,7 +460,13 @@ let make = () => {
                             permission=userPermissionJson.merchantDetailsManage>
                             <HSwitchSettings />
                           </AccessControl>
-                        | list{"account-settings", "profile"} => <HSwitchProfileSettings />
+                        | list{"account-settings", "profile", ...remainingPath} =>
+                          <EntityScaffold
+                            entityName="ConfigurePMTs"
+                            remainingPath
+                            renderList={() => <HSwitchProfileSettings />}
+                            renderShow={value => <ModifyTwoFaSettings />}
```

Comment:
wrap the <ModifyTwoFaSettings /> with the featureflag totp

Reply:
Wrapped  <ModifyTwoFaSettings /> with featureflag as the whole component cannot be wrapped